docs/src/content/docs/components/tabs.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: Tabs
+description: Learn how to create tabbed interfaces in Starlight.
```

Comment:
```suggestion
description: Learn how to create tabbed interfaces in Starlight to display equivalent information closely.
```

Reply:
Not sure about the "closely" here. Maybe something like "...to group equivalent information"?